lua/entities/gmod_wire_expression2/core/string.lua

```diff
@@ -38,14 +38,14 @@ end)
 
 registerOperator("fea", "nss", "", function(self, args)
 	local keyname, valname = args[2], args[3]
-
+	local sub = utf8.sub
```

Comment:
Nothing else in the string library assumes encoding, probably shouldn't here either. Also you would need to use `uft8.len` as well.